### DISCUSSION TITLE
Generate static CSS for custom patterns.

### DISCUSSION THREAD

**Discussion by soroush96** [chakra-ui/panda#1882] [Ideas]
### Description

Currently the styles made by custom patterns are generated if they have been previously used in code, It would be nice to have all the possible styles generated before even being used in code.

### Problem Statement/Justification

The styles that are given to a pattern with values of tokens, create classes of the said style but it does not generate the corresponding style of that class until the said class is mentioned in consumer's code.

### Proposed Solution or API

 It would be a good idea to be able to generate these styles before we analyze consumer's code when using patterns.

**Comment by segunadebayo** [comment]
This was added in a recent PR here https://github.com/chakra-ui/panda/pull/1861

It'll be published pretty soon.
